framework/include/mfem/problem/MFEMProblem.h

```diff
@@ -199,6 +226,24 @@ class MFEMProblem : public ExternalProblem
   Moose::FEBackend feBackend() const override { return Moose::FEBackend::MFEM; }
 
   std::string solverTypeString(unsigned int solver_sys_num) override;
+  bool getMeshChanged() const { return _problem_data._mesh_changed; }
+  void setMeshChanged(bool ch) { _problem_data._mesh_changed = ch; }
+
+  void updateAfterRefinement();
+  void updateFESpaces();
+  void uniformRefinement(int num_refinements);
+
+  void SetUpAMR();
+
+  //! Return true when it's time to stop
+  void HRefine();
+  //! Return true when it's time to stop
+  void PRefine();
+
+  bool UseHRefinement() const;
+  bool UsePRefinement() const;
+
+  bool UseAMR() const { return _problem_data._use_amr; }
```

Comment:
Let's use camelCase instead of PascalCase wherever possible (which usually means whenever we're not overriding MFEM's API).